### PR TITLE
feat(social-media-follow): add component - INNO-1221

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

Add component social media follow, according to [specifications](https://webgate.ec.europa.eu/CITnet/confluence/display/NEXTEUROPA/Social+Media+Follow+-+Design+Specifications)

Also
- allows multiple icons in links
- add missing sizes for icons

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 6614171b64c4e8f02916a11f7d7125d8b53a9e0b

https://deploy-preview-925--europa-component-library.netlify.com

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 6e1206ffaafe8a220618bbb9a158b95b88a61526

https://deploy-preview-925--europa-component-library.netlify.com

File: src/systems/ec/implementations/react/packages/link/Link.jsx [hunk 1]
```diff
@@ -14,19 +14,28 @@ const Link = ({
   className,
   ...props
 }) => {
+  const iconMarkup =
+    icon.constructor === Array ? (
```
Comment (yhuard) [inline src/systems/ec/implementations/react/packages/link/Link.jsx]: `Array.isArray(icon)` is better 
— https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray

Comment (emeryro) [reply src/systems/ec/implementations/react/packages/link/Link.jsx]: it seems to be slower... I followed the comment here: https://stackoverflow.com/a/26633883

File: src/systems/ec/implementations/react/packages/social-media-follow/SocialMediaFollow.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (yhuard) [inline src/systems/ec/implementations/react/packages/social-media-follow/SocialMediaFollow.jsx]: Please remove this comment :)

**Review by degliwe — APPROVED** [review]

**Pull request merged**